app/models/mixins/relationship_mixin.rb

```diff
@@ -565,11 +618,14 @@ def add_children(*child_objs)
   alias_method :add_child, :add_children
 
   def parent=(parent)
+    return call_ancestry_method(:parent=, parent) if use_ancestry?
```

Comment:
~~Are most of the blocks of code assuming that the `parent=` will update this value instantly?~~
Nice, so genealogy uses `update!` so this does not need to concern itself with the change in `save` behavior

to be honest, `parent=` is the one place I find the most concerning
especially when the parent node is not archived but rather removed (archived is fine)

Reply:
this shows a warning. I think it is fine

the alternative will also show a warning

```suggestion
    call_ancestry_method(:parent=, parent) and return if use_ancestry?
```